evidence/services/services_utils.go

```diff
@@ -0,0 +1,30 @@
+package services
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+func (es *EvidenceService) IsEvidenceSupportsProviderId() bool {
+	// providerId is supported from evidence version XXX
+	// get evidence version API was added afterwards so we will check that the API returns 200 OK
+	// and not 404 Not Found
+
+	requestFullUrl, err := url.Parse(es.GetEvidenceDetails().GetUrl() + "api/v1/system/version")
+	if err != nil {
+		return false
+	}
+
+	httpClientDetails := es.GetEvidenceDetails().CreateHttpClientDetails()
+	httpClientDetails.SetContentTypeApplicationJson()
+
+	log.Debug("Checking evidence version: ")
```

Comment:
you ar eonly checking if api exists, you dont want to check the actual version ?

Reply:
Since we exposed the version API late (For this task specifically and overall for CLI), we have a situation where providerId was added before the version API. 
I'll add a more detailed warning containing the version when it's released in SH.